### PR TITLE
[FIX] web: search view after url loaded

### PR DESCRIPTION
**Pull request by Polymorphe57**

### CODE REVIEW

File: addons/web/static/src/webclient/actions/action_service.js [hunk 1]
```diff
@@ -706,9 +710,10 @@ function makeActionManager(env) {
                 currentController.getGlobalState() // what if this = {}?
             );
         }
-        if (controller.action.globalState) {
+        if (!controller.isLazy && controller.action.globalState) {
```
Comment (kebeclibre) [inline addons/web/static/src/webclient/actions/action_service.js:713]: not usefull anymore

File: addons/web/static/src/webclient/actions/action_service.js [hunk 2]
```diff
@@ -399,11 +399,15 @@ function makeActionManager(env) {
                 return viewSwitcherEntry;
             });
         const context = action.context || {};
+        let groupBy = context.group_by;
```
[the remark this answers is not in the document]
Comment (aab-odoo) [reply addons/web/static/src/webclient/actions/action_service.js:402]: I do not completely agree. The action service generates view props, which are part of the public API of views (not really an implementation detail). It's its responsibility to generate correct props.

**Comment by aab-odoo** [comment]
robodoo rebase-ff

**Comment by aab-odoo** [comment]
robodoo r+

**Comment by aab-odoo** [comment]
robodoo r-

**Comment by aab-odoo** [comment]
robodoo r+

**Pull request closed without merging**